ecr_api.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, app):
         self.password = ''
         #self.authenticated = False
 
-    def __call__(self, environ, start_response):
+    def __call__(self, environ, start_response): # pragma: no cover
```

Comment:
It would be nice if this `pragma` was removed and instead we tested our DB database access error condition by doing a mock on the `requests.post` call (both positive and negative scenarios).